static/src/javascripts/projects/common/modules/accessibility/helpers.js

```diff
@@ -0,0 +1,26 @@
+define([
+    'fastdom',
+    'common/modules/accessibility/main',
+    'common/utils/$'
+], function (
+    fastdom,
+    accessibility,
+    $
+) {
+    function hideFlashingImages(callback) {
+        if (!accessibility.isOn('flashing-images')) {
```

Comment:
if isOn flashing images  then remove flashing images.  some reverse logic is confusing me there!